src/shared-components/dropdown/desktopDropdown.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+
+import OffClickWrapper from '../offClickWrapper';
+import { ChevronIcon } from '../../icons';
+import {
+  SelectOuterContainer,
+  selectInputStyle,
+  IconContainer,
+  SelectOptionsContainer,
+  SelectOption,
+} from './style';
+
+const DesktopDropdown = ({
+  value,
+  options,
+  textAlign,
+  currentOption,
+  closeDropdown,
+  onOptionClick,
+  onSelectClick,
+  isOpen,
+}) => (
+  <OffClickWrapper onOffClick={closeDropdown} css={css`width: 100%;`}>
+    <SelectOuterContainer textAlign={textAlign}>
+      <div id="select-input-box" onClick={onSelectClick}>
+        <div css={selectInputStyle({ textAlign })}>
+          {currentOption && currentOption.label}
+        </div>
+        <IconContainer>
+          <ChevronIcon width={10} height={10} rotate={isOpen ? 90 : 0} />
+        </IconContainer>
+      </div>
+
+      <SelectOptionsContainer isOpen={isOpen}>
+        {options.map(option => (
+          <SelectOption
+            key={option.value}
+            value={option.value}
+            selected={value === option.value}
+            disabled={option.disabled}
+            onClick={onOptionClick}
+            data-dusk-id={`selectOption-${option.value}`}
```

Comment:
we should try to remove the dusk id from Radiance since it's specific to the Laravel app. The way we've handled this before is to pass along `...rest` after destructuring `props` to a component. I think in this case, we can destructure the option with `const { value, disabled, label, ...rest } = option` and pass along `...rest` to the `SelectOption` component